composer.json

```diff
@@ -16,7 +16,8 @@
     "ext-curl": "*",
     "ext-dom": "*",
     "ext-libxml": "*",
-    "apimatic/core-interfaces": "~0.1.4",
+    "psr/log": "^1.1.4 || ^2.0.0 || ^3.0.0",
+    "apimatic/core-interfaces": "dev-test-logging",
```

Comment:
@sufyankhanrao I will change this to `~0.1.5`, after releasing core-interfaces version, before merging this PR.